modules/docs/lib/InformationHighlight/index.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import {createContainer} from '@workday/canvas-kit-react/common';
+import {createModifiers, createStyles, cssVar} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+
+import {Base, BaseProps} from './Base';
+import {Heading} from './Heading';
+import {Body} from './Body';
+import {Icon} from './Icon';
+import {Link} from './Link';
+import {useInformationHighlightModel} from './modelHook';
+
+type Variant = 'emphasis' | 'caution' | 'attention';
```

Comment:
Is `emphasis` considered the default or is there a "default" variant?

Reply:
Emphasis happens to be the default variant set in the model. But there's no reason for it to be default.

Reply:
Do we think that there could be any use case for a "default" variant. Something that is based on `blackPepper` or something like that?

Reply:
There's nothing in the spec for that. This is borrowing a similar style to link out to docs until we're ready to have a real Info Highlight.